examples/common.py

```diff
@@ -13,9 +13,6 @@
     "APTOS_FAUCET_URL",
     "https://faucet.devnet.aptoslabs.com",
 )
-INDEXER_URL = os.getenv(
-    "APTOS_INDEXER_URL",
-    "https://api.devnet.aptoslabs.com/v1/graphql",
-)
+INDEXER_URL = os.getenv("APTOS_INDEXER_URL")
```

Comment:
why remove this?

Reply:
This first looks into `APTOS_INDEXER_URL`, but in the case user is running localnet without the `--with-indexer-api` (and without `APTOS_INDEXER_URL`) this default value will give us the devnet indexer url, when we're running local tests without an indexer.

Test cases using indexer like `test_transfer_coin` fails if we keep this default value.